website/integrations/services/freshrss/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: FreshRSS
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is FreshRSS
+
+> FreshRSS is a self-hosted RSS feed aggregator.
+>
+> -- https://github.com/FreshRSS/FreshRSS
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `service.company` is the FQDN of the FreshRSS install.
+-   `port` is the port on which the FreshRSS install is running (usually 443)
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik Configuration
+
+In Authentik, create an _OAuth2/OpenID Provider_ under _Applications > Providers_.
+
+** Protocol Settings **
+_Client Type_ : _Confidential_
+
+:::note
+Take note of the `Client ID` and `Client Secret`, you'll need them later.
+:::
+
+_Redirect URIs/Origins_ :
+
+-   `https://service.company/i/oidc/`
+-   `https://service.company:port/i/oidc`
```

Comment:
Can we assume in this document that freshrss is running behing a reverse proxy and thus that `:port` is not required in the URL? (We can then also remove it from the above placeholders)

Reply:
Hello. @rissson. Unfortunately we can't, we need to add the :port alternative, even behind a reverse proxy, that's why I mentioned it